### ISSUE TITLE
Rides

### ISSUE THREAD

**Issue by harrislapiroff** [littleweaver/django-brambling#194]
Should we offer ride coordinator functionality, too? It's somewhat analogous to housing stuff and it actually does seem strange to me to have one and not the other.

**Comment by harrislapiroff** [comment]
Here's an example of an org ad hocking a solution to this problem: https://sites.google.com/a/oberlin.edu/dandelionromp/home

It could be as simple as providing a rideboard like that where people can post information. I think even having it integrated into the same tool as registration would make it more likely to be used and therefore easier for organizers. Might be worth doing some research with people who have done rides coordination for events, though.

**Comment by melinath** [comment]
(Basically it's just a simple google spreadsheet.) +1 that something this simple could probably be easy for people to use, and could be great if we implemented it right. Big +1 on doing research. I'd love to treat this similar to housing; if the redesigned checkout process & form management interfaces could accommodate this kind of thing it would be great.

![screen shot 2016-04-20 at 9 52 08 am](https://cloud.githubusercontent.com/assets/299979/14683253/9bd6b39e-06de-11e6-9ff1-86f4e2e0b90f.png)
![screen shot 2016-04-20 at 9 52 29 am](https://cloud.githubusercontent.com/assets/299979/14683252/9bd6855e-06de-11e6-9ab9-3014fd57dbd9.png)

**Comment by harrislapiroff** [comment]
Yeah. With both this and housing, I'm increasingly interested in the idea of separating them from the registration process. Like maybe you do your registration and pay and then you get access to an "attendees area" where you can do things like request housing, arrange rides, maybe someday socialize with other attendees pre-event...

But yeah, research would be good.

**Comment by melinath** [comment]
I think that for housing and rides we would probably still want to include them in the registration process so that people who needed it didn't miss it, BUT also have a very clear way to update your status post-registration.

**Comment by melinath** [comment]
Talked with an organizer this weekend who was also really into the idea of ride share handling, even if it were really simple. Their take: Right now people are trying to do it anyway inside the Facebook event, and it's a pain in the ass and doesn't work very well.
